paywall/src/unlock.js/startup.js

```diff
@@ -0,0 +1,38 @@
+import { makeIframe, addIframeToDocument } from './iframeManager'
+import setupPostOffices from './setupPostOffices'
+import dispatchEvent from './dispatchEvent'
+
+export default function startup(window) {
+  try {
+    // this is a cache for the time between script startup and the full load
+    // of the data iframe. The data iframe will then send down the current
+    // value, overriding this. A bit later, the blockchain handler will update
+    // with the actual value, so this is only used for a few milliseconds
+    const locked = JSON.parse(
```

Comment:
I think we may want to let the publisher decide of what is the "default" state when none is set.

Reply:
For another PR?